frontend/src/pages/PassingTest/PassingTestContainer.tsx

```diff
@@ -1,23 +1,45 @@
-import React, { useState } from 'react';
-import { useParams } from 'react-router';
+import React, { useState, useEffect } from 'react';
+import { useParams, useNavigate } from 'react-router';
 
+import timeIsOverDialogToggle from 'hooks/useToggle';
 import { MAX_STAGE_INITIAL, MIN_STAGE, STAGE_CHANGE } from 'constants/test';
+import { PATHS } from 'constants/routes';
 import { useSendTestResult, useGetCourseTest } from 'api/test';
 import { useFinishClientCourse, useGetClientCourseInfo } from 'api/myCourses';
 
 import PassingTest from './PassingTest';
 import TestResult from './TestResult';
 import ConfirmLeavePage from './ConfirmLeavePage';
+import ConfirmTimeIsOver from './ConfirmTimeIsOver';
 
 const PassingTestContainer: React.FC = () => {
   const params = useParams();
+  const naviagteTo = useNavigate();
 
   const { data: courseTestData, isLoading } = useGetCourseTest({ courseId: params.courseId });
   const { data: clientCourseResponse } = useGetClientCourseInfo(params.courseId);
 
   const courseTest = courseTestData?.length ? courseTestData[0].test : undefined;
   const maxStage = courseTest ? courseTest.questions.length : MAX_STAGE_INITIAL;
 
+  const [isOpenedTimeIsOverDialog, seTIsOpenedTimeIsOverDialog] = timeIsOverDialogToggle(false);
```

Comment:
```suggestion
  const [isOpenedTimeIsOverDialog, setIsOpenedTimeIsOverDialog] = timeIsOverDialogToggle(false);
```

Also, maybe we can rename the variables to be more descriptive? The meaning is not entirely clear

Reply:
done